medusa/server/web/config/post_processing.py

```diff
@@ -213,19 +220,13 @@ def isNamingValid(pattern=None, multi=None, abd=False, sports=False, anime_type=
             return 'invalid'
 
     @staticmethod
-    def isRarSupported():
-        """
-        Test Packing Support:
-            - Simulating in memory rar extraction on test.rar file
-        """
-
+    def isRarSupported(unrar_tool):
```

Comment:
Here `config/postProcessing/isRarSupported` is a route. The same as `config/postProcessing/savePostProcessing` and `home/testSABnzbd` these are all in camelCase. So this is correct, or at least following the current convention.